solutions/swb-reference/integration-tests/tests/multiStep/userProject.test.ts

```diff
@@ -46,7 +46,7 @@ describe('multiStep user to project integration test', () => {
       const { data: user } = await adminSession.resources.users.create({
         firstName: 'Project',
         lastName: role,
-        email: `success-user-to-project-user-${uuidv4()}@simulator.amazonses.com`
+        email: `success+user-to-project-user-${uuidv4()}@simulator.amazonses.com`
```

Comment:
why the switch?

Reply:
- means nothing, while + will redirect emails to `success@simulator.amazonses.com` . I recently got email about large percentage of bouncing emails for cognito users when was running tests, but didn't realize that `success@simulator.amazonses.com` is actually exist